Dockerfile-test

```diff
@@ -0,0 +1,26 @@
+# import simd from ibc-go
```

Comment:
Don't have much context here but why is this from `ibc-go`? Isn't it from `cosmos-sdk`?

Reply:
Thanks, I changed this during dev but didn't change the comment :)